packages/cursorless-engine/src/processTargets/modifiers/scopeHandlers/ScopeHandlerFactoryImpl.ts

```diff
@@ -50,9 +53,11 @@ export class ScopeHandlerFactoryImpl implements ScopeHandlerFactory {
       case "document":
         return new DocumentScopeHandler(scopeType, languageId);
       case "oneOf":
-        return new OneOfScopeHandler(this, scopeType, languageId);
+        return OneOfScopeHandler.create(this, scopeType, languageId);
       case "paragraph":
         return new ParagraphScopeHandler(scopeType, languageId);
+      case "custom":
+        return scopeType.scopeHandler;
```

Comment:
For convenience, you can pass in a custom scope type.  It's just as easy for the caller to grab the scope handler themselves, but the uniformity of just calling `scopeHandlerFactory.create` can make their life easier